infrastructure/server-setup/production.ini

```diff
@@ -7,6 +7,12 @@
 ;
 ; Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
 
+[all:vars]
+; This configuration variable blocks all access to the server, including SSH, except from the IP addresses specified below.
+; This should always be set when configuring a production server if there is no other firewall in front of the server.
+; SSH and other services should never be exposed to the public internet.
+only_allow_access_from_addresses=165.22.205.62,165.22.110.53
```

Comment:
Move this to staging

Reply:
If the QA server is also behind the VPN, and if the VPN goes down then this would be not necessary.  We could use SSH with public and private key to access the VPN server. So perhaps we can remove this when we finish testing.